presctl/cmd/entities/recognizers.go

```diff
@@ -0,0 +1,43 @@
+package entities
+
+import (
+	"fmt"
+
+	"github.com/spf13/viper"
+)
+
+const recognizerURLFormat = "http://%s:%s/api/v1/analyzer/recognizers/%s"
+
+func constructRecognizerURL(recognizerName string) string {
+	var ip = viper.GetString("presidio_ip")
```

Comment:
Seems that you can re-use the ip and port code also in templates.
Will it make sense to put it common.go as well?

Reply:
done